performance-test/support-files/request_id.txt

```diff
@@ -0,0 +1,3199 @@
+request_trn_id
+3707c997-2ffe-4138-9a6d-e1f20dadb648
+98f06253-cca9-4c6f-be09-08750ac4a4fc
```

Comment:
Do we need to commit this file, is this not part of setup or preparation ?

Reply:
Removing this file, as it is not required.

Reply:
Reviewed. Request_id file has been removed